handbook/engineering/hiring/software-engineer-campaigns.md

```diff
@@ -0,0 +1,75 @@
+
+## Not sure if this is you?
+
+We want a diverse, global team, with a broad range of experience and perspectives. If this job sounds great, but you’re not sure if you qualify, apply anyway! We carefully consider every application, and will either move forward with you, find another team that might be a better fit, keep in touch for future opportunities, or thank you for your time.
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook).
+
+## Interview process
+
+<!-- 1. You [apply here](#todo). --><!-- This is commented out because I will be sending some outbound messages before accepting applications. -->
+
+1. The **Hiring Manager** reviews all the information you provided on your application to determine if you meet our qualifications for this role (if there is another open role we think you would be better qualified for, we will let you know).
+1. A **Recruiter** sets up a 30 minute call to learn more about what you are looking for, tell you about Sourcegraph, and answer any questions that you have.
+1. We set up a 1 hour interview with the **Hiring Manager**, who will tell you more about the role in the team. They will ask you about your past work and accomplishments and assess your suitability for the role.
+1. You complete a 3 hour [coding exercise](software-engineer-coding-exercise.md#frontend-coding-exercise) that we designed to test your technical ability and the responsibilities of the role as [listed above](#your-responsibilities).
+   - _Note: while we do not disqualify candidates who do not have React and TypeScript experience from taking the exercise, you will be required to use these for the exercise._
+1. An **Engineer** and a **Designer** will review and grade your submission. We will send you our review along with some follow-up questions.
+1. We schedule the following additional interviews, in no particular order, across multiple days:
+   - 1 hour **Team collaboration:** We ask you about how you work and communicate in a team setting, how you handle tricky situations, decisions you made, and what you'd do differently today.
+   - 1 hour **Code collaboration:** You pair with an engineer on a small task to ensure you would be a good fit for this particular team. Another engineer will observe, to reduce bias.
```

Comment:
The 3 hour exercise is about technical proficiency, while the 1 hour pairing is about "pairing chemistry" with the team: Does this engineer pair well with members of this team? (As this comes after the async exercise, technical proficiency has already been established and is assumed.)

I don't have this documented anywhere outside of this posting. Where would you suggest it live?

Reply:
> Where would you suggest it live?

Ideally on a separate page in the handbook that both candidates and interviewers can read to understand what will happen.

Reply:
I am particularly concerned about the vague "ensure you would be a good fit for this particular team". Instead of saying "fit" enumerate what we are actually looking for.

Reply:
Thanks, you made a great point. I added a page to explain just what it is that I'm looking for in a pairing exercise (and what I'm not looking for), and removed language around "fit".